raiden/ui/app.py

```diff
@@ -273,6 +273,8 @@ def run_app(
         config['environment_type'] = Environment.DEVELOPMENT
 
     environment_type = config['environment_type']
+    print(f'Raiden is running in {environment_type.value.lower()} mode')
```

Comment:
Can't you use `click.[s]echo` here, or some `log`, instead of `print`?

Reply:
Wasn't working well with the CLI tests. And we already use print to denote some things like "You are connected to NETWORKNAME".